Main.hs

```diff
@@ -4,22 +4,39 @@ import Control.Monad (unless)
 import Text.Printf (printf)
 
 short :: [a] -> Bool
-short = undefined 
+-- short ls = length ls < 3
+short (a : b : c : xs) = False
```

Comment:
No need to introduce variable names when you are not using them in the body. Use wildcards instead.